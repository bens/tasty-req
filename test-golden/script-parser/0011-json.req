1: GET /
---
{ "a": null,
  "b": true,
  "c": false,
  "d"
:
"foo"
,
  "e": "abc\nxyz\tuvw",
  "f": [1,2, 
3 ],
  "g": { "x": 
null, 
 	"y": true
, "z"
: false
 },
  }
