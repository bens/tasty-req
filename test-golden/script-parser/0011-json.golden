1: GET /
---
{"g": {"z": false,
       "y": true,
       "x": null},
 "f": [1,
       2,
       3],
 "e": "abc\nxyz\tuvw",
 "d": "foo",
 "c": false,
 "b": true,
 "a": null}
